### ISSUE TITLE
Support model pickling

### ISSUE THREAD

**Issue by rstrudel** [stack-of-tasks/pinocchio#957]
Hi,

When I try to dump a model using pickle, I get the following error:
`RuntimeError: Pickling of "pinocchio.libpinocchio_pywrap.Model" instances is not enabled (http://www.boost.org/libs/python/doc/v2/pickle.html)`

Here is a short snippet to reproduce the error:
```
import pickle as pkl
import pinocchio as pin

model = pin.Model()
pkl.dump(model, open('model.pkl', 'wb'))
```

Would it be possible to include this feature ?
Thanks,

Robin

**Comment by jcarpent** [comment]
Solved by #958.

**Comment by joao-pm-santos96** [comment]
Hi! Sorry to "re-open" this already closed issue but why can we pickle pin.Model() but not pin.GeometryModel()?

Thank you!
